test/test-clusters/src/main/java/org/elasticsearch/test/cluster/local/FipsEnabledClusterConfigProvider.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+package org.elasticsearch.test.cluster.local;
+
+import org.elasticsearch.test.cluster.util.resource.Resource;
+
+public class FipsEnabledClusterConfigProvider implements LocalClusterConfigProvider {
+
+    @Override
+    public void apply(LocalClusterSpecBuilder builder) {
+        if (isFipsEnabled()) {
+            builder.configFile(
+                "fips_java.security",
+                Resource.fromClasspath(isOracleJvm() ? "fips/fips_java_oracle.security" : "fips/fips_java.security")
+            )
+                .configFile("fips_java.policy", Resource.fromClasspath("fips/fips_java.policy"))
+                .configFile("cacerts.bcfks", Resource.fromClasspath("fips/cacerts.bcfks"))
+                .systemProperty("java.security.properties", "${ES_PATH_CONF}/fips_java.security")
```

Comment:
I think this property value should have additional `=` char as in [elasticsearch.fips.gradle](https://github.com/elastic/elasticsearch/blob/2ca52e530a1e799361f22925f480455621a2d2f3/build-tools-internal/src/main/groovy/elasticsearch.fips.gradle#L63) file:
```suggestion
                .systemProperty("java.security.properties", "=${ES_PATH_CONF}/fips_java.security")
```

This was intentional in order to get the double equals sign when specifying system property, i.e. it should look like this:  

- `-Djava.security.properties==${ES_PATH_CONF}/fips_java.security`

The `==` means that just the specified security property file will be used.

Reply:
Thanks. I thought that might have been a typo, however I didn't notice any behavior difference between the two.